src/coresyftools/coresyftools/packager.py

```diff
@@ -49,13 +52,17 @@ def _check_tool_directory_structure(self):
             raise TargetDirectoryNotFoundException()
         if not exists(join(self.tool_dir, 'run')):
             raise MissingRunFileException()
-        if not exists(join(self.tool_dir, 'manifest.json')):
+        manifest_files = glob.glob(join(self.tool_dir, '*manifest.json'))
+        if not manifest_files:
             raise MissingManifestFileException()
+        if len(manifest_files) > 1:
+            raise MultipleManifestFileException()
         if not exists(join(self.tool_dir, 'examples.sh')):
             raise MissingExamplesFileException()
     
     def _read_manifest(self):
-        self.manifest = get_manifest(join(self.tool_dir, 'manifest.json'))
+        manifest_file_name = glob.glob(join(self.tool_dir, '*manifest.json'))
```

Comment:
Since this repeats line 55, I would put it in a new function `find_manifest_file` or something like that.